index.js

```diff
@@ -0,0 +1,58 @@
+require('dotenv').config()
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+app.use(bodyParser.json())
+const freeclimbSDK = require('@freeclimb/sdk')
+
+const port = process.env.PORT || 3000
+const accountId = process.env.ACCOUNT_ID
+const apiKey = process.env.API_KEY
+const fromNumber = process.env.FREECLIMB_NUMBER
+const baseServer = new freeclimbSDK.ServerConfiguration(process.env.API_SERVER || "https://www.freeclimb.com/apiserver")
+const freeclimbConfig = freeclimbSDK.createConfiguration({ baseServer, accountId, apiKey })
+const apiInstance = new freeclimbSDK.DefaultApi(freeclimbConfig);
+
+app.post('/incomingSms', (req, res) => {
+  const { from: userPhoneNumber } = req.body
```

Comment:
Instead of using request.from, we should have another environment variable for the to variable:

https://github.com/FreeClimbAPI/PHP-SMS-Quickstart/pull/3#discussion_r1585580921